easybuild/easyblocks/generic/intelbase.py

```diff
@@ -112,6 +112,7 @@ def extra_options(extra_vars=None):
         extra_vars = EasyBlock.extra_options(extra_vars)
         extra_vars.update({
             'license_activation': [ACTIVATION_LIC_SERVER, "License activation type", CUSTOM],
+            'licensed': [True, "Set to False if Intel software does not require a runtime licence", CUSTOM],
```

Comment:
Is this name potentially misleading, of course it is still licenced but doesn't require a runtime licence, how about `runtime_license`?